users.go

```diff
@@ -145,3 +147,16 @@ func (c *Client) UpdatePassword(accessToken, realm, currentPassword, newPassword
 	var m = map[string]string{"currentPassword": currentPassword, "newPassword": newPassword, "confirmation": confirmPassword}
 	return c.post(accessToken, nil, url.Path(accountPasswordPath), url.Param("realm", realm), body.JSON(m))
 }
+
+// GetAccount updates the user's informations
```

Comment:
// GetAccount returns/provides (?) the user's information. (no s)

Reply:
thks